projects/view/client/rtProject/rtProjectInfoSection.css

```diff
@@ -0,0 +1,215 @@
+.rtProjectInfoSection.span_3_of_12:nth-child(2) .circle{
+	  -webkit-animation-delay: 0.2s; /* Chrome, Safari, Opera */
+    animation-delay: 0.2s;
+}
+.rtProjectInfoSection .span_3_of_12:nth-child(3) .circle{
+	  -webkit-animation-delay: 0.3s; /* Chrome, Safari, Opera */
+    animation-delay: 0.3s;
+}
+.rtProjectInfoSection .span_3_of_12:nth-child(4) .circle{
+	  -webkit-animation-delay: 0.4s; /* Chrome, Safari, Opera */
+    animation-delay: 0.4s;
+}
+.rtProjectInfoSection .span_3_of_12:nth-child(5) .circle{
+	  -webkit-animation-delay: 0.6s; /* Chrome, Safari, Opera */
+    animation-delay: 0.6s;
+}
+.rtProjectInfoSection .circle h3{
+	line-height: 200px;
+	color:#fff;
+}
+.rtProjectInfoSection .circle .text{
+	text-align: center;
+	color:#1F225D;
+}
+.rtProjectInfoSection .projectDetail{
+	overflow: hidden;
+	position: relative;
+	max-width: 900px;
+	margin-bottom: 40px;
+	margin-top: 40px;
+}
+
+.rtProjectInfoSection .factorWrapper{
+	background-color: #f5f5f5;
+}
+.rtProjectInfoSection .factors{
+	max-width: 1014px;
+	text-align: center;
+	padding-top: 40px;
+	padding-bottom: 40px;
+}
+@media only screen and (max-width: 1024px) {
+.rtProjectInfoSection 	.factors .span_3_of_12{
+		width: 49%;
+	}
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection 	.factors .span_3_of_12{
+		width: calc(100% - 20px);
+		padding-left: 20px;
+	}
+}
+.rtProjectInfoSection .factors p{
+	color:#1F225D;
+}
+/**************** timeline owners ********************/
+.rtProjectInfoSection .timeline{
+	margin-top: 80px;
+	position: relative;
+	overflow: hidden;
+}
+.rtProjectInfoSection .timeline #line{
+	position: absolute;
+	top:20px;
+	bottom:20px;
+	left:calc(50% - 2px);
+	width: 4px;
+	background-color: #F0F0F0;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection 	.timeline #line{
+		left:23px;
+		}	
+}
+.rtProjectInfoSection .timeline-object{
+	width: 100%;
+	max-width: 900px;
+	position: relative;
+}
+.rtProjectInfoSection .timeline-object .span_5_of_12{
+	opacity: 0;
+	width: 10%;
+	-webkit-transition: all 0.2s cubic-bezier(0.44, 0.88, 0.85, 1.11);
+	-moz-transition: all  0.2s cubic-bezier(0.44, 0.88, 0.85, 1.11);
+	transition: all  0.2s cubic-bezier(0.44, 0.88, 0.85, 1.11);
+}
+.rtProjectInfoSection .timeline-object.active .span_5_of_12{
+	opacity: 1;
+	width: 41.08%;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection	.timeline-object.active .span_5_of_12{
+		width: calc(100% - 60px);
+		margin-left: 60px
+	}
+}
+.rtProjectInfoSection .timeline-object:nth-child(2n + 2) .col{
+	float: right;
+}
+@media only screen and (max-width: 480px) {
+.rtProjectInfoSection	.timeline-object:nth-child(2n + 2) .col{
+		float: left;
+	}
+}
+@media only screen and (min-width: 480px) {
+.rtProjectInfoSection 	.timeline-object:nth-child(2n + 2).active .span_5_of_12::after{
+			border-radius: 100px;
+			content: " ";
+	        display: block;
+	        background: #EC2870;
+	        height: 100px;
+	        width: 100px;
```

Comment:
Check indentation and whitespace. Two spaces per indent. Use relative units such as `em` or `rem` for layout related style rules.